src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -3400,7 +3424,10 @@ public enum DarknessAblaze implements LogicCardInfo {
             assert my.deck : "You have no cards left in your deck"
           }
           onAttack {
-            my.deck.search("Put Energy Card into your hand",cardTypeFilter(ENERGY)).showToOpponent("Opponent's chosen Energy card").moveTo(my.hand)
+            if (my.deck) {
```

Comment:
Won't this always be true because of the `attackRequirement`?

Reply:
Ah nevermind I found it.  You were considering the case where the attack was copied

Reply:
Not if the attack is copied by another attack/ability.